### PR TITLE
Standardise deprecations

### LINKED ISSUE #1643: Standardise deprecation warnings
Deprecation warnings are not consistently formatted in Horace. 
```
HORACE:cut_sym:deprecated
HORACE:deprecated:invalid_argument
MAGNETICIONS:depricated_method
```

### PR DESCRIPTION
**Pull request by oerc0122**
Believed to be a complete set of deprecation changes

Fixes #1643

### CODE REVIEW

File: horace_core/symop/@Symop/Symop.m [hunk 1]
```diff
@@ -237,7 +237,17 @@ function disp(obj)
...
-                    end
+                    %% TODO non-aligned ez/ey not supported
+                    % ez_new = obj.R * proj.ez(:);
+                    % ey_new = obj.R * proj.ey(:);
+
+                    %                 offset_new = proj.offset(:);
+                    %                 offset_new(1:3) = obj.transform_vec(offset_new(1:3));
+                    %
+                    %                 proj.offset = offset_new;
+                case 'spher_proj'
```
Comment (cmarooney-stfc) [inline horace_core/symop/@Symop/Symop.m:248]: this looks like  a clash or duplicate with line 236

**Review by cmarooney-stfc — CHANGES_REQUESTED** [review]

File: horace_core/symop/@Symop/Symop.m (hunk 1 above)
Comment (oerc0122) [reply horace_core/symop/@Symop/Symop.m:248]: Note the lack of an `e`

Comment (oerc0122) [reply horace_core/symop/@Symop/Symop.m:248]: Nope, you're right. Merge/rebase/whatever error

Comment (oerc0122) [reply horace_core/symop/@Symop/Symop.m]: That entire thing shouldn't even be there, the deprecation would be caught when the objects were instantiated. Removed

**Review by cmarooney-stfc — APPROVED** [review]

**Pull request merged**